docs/01-Introduction.md

```diff
@@ -1,51 +1,62 @@
+and links them with corresponding behavioral data in LORIS. Scans can be viewed 
+and quality controlled in the LORIS front end via web browser, facilitating 
+collaboration between radiologists, clinicians and researchers.
+
+### 1.2 How does LORIS-MRI work? 
+ 
+![user_story](images/user_story.png)
 
-### 1.2 How does LORIS-MRI work?  ![user_story](images/user_story.png)
 LORIS-MRI allows multiple ways to upload scans, but typically, users
-upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder to Imaging
-Uploader.  LORIS requires that the uploaded file name follow the naming
-convention PSCID_CANDID_VISIT-LABEL. 
+upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder via the Imaging
+Uploader module that should be composed of **only** DICOM files. LORIS requires 
+that the uploaded file name follow the naming convention 
+`PSCID_CANDID_VISIT-LABEL`. 
 In addition, all DICOM datasets uploaded via the Imaging Uploader or 
 transferred on the LORIS-MRI server must be free of any identifying 
-information (e.g. patient name). A tool can be provided to the sites to 
-facilitate deidentification. Please contact the LORIS team for details.
+information (*e.g.* patient name). A tool can be provided to the sites to 
+facilitate de-identification. Please contact the LORIS team for details.
 
 The LORIS-MRI pipeline starts once the scans are uploaded to the server.
 The pipeline can start automatically if the autolaunch configuration is
-set, otherwise manual intervention is required by a developer with backend 
-access to the server. More details about these options and the scripts
-needed to run the pipeline can be found [here](05-PipelineLaunchOptions.md). 
+set, otherwise a back-end administrator can manually run the pipeline. 
+These options and scripts are detailed in the 
+[Pipeline Triggering Options documentation](05-PipelineLaunchOptions.md). 
 
-Insertion progress can be tracked by the user through the Log Viewer in 
-Imaging Uploader, where descriptive messages can be consulted.
+Insertion progress can be tracked by the user through the Log Viewer in the
+Imaging Uploader module, where descriptive messages can be consulted.
 The output of the main key steps in the insertion progress can also be consulted 
-through the LORIS DICOM archive module for successfully archived DICOM datasets, 
-Imaging Browser for MINC-generated files that pass the study-defined MRI protocol, 
-and 3D or 4D navigation of these MINC files in BrainBrowser. More details on the
-capabilities of BrainBrowser can be found [here.][5]
+through:
+ - the LORIS DICOM Archive module for successfully archived DICOM datasets`*` 
+ - the Imaging Browser module for MINC files (generated from DICOM) that pass the 
+study-defined MRI protocol 
+ - BrainBrowser using 3D or 4D navigation of these MINC files. More details on 
+BrainBrowser's capabilities can be found [here.][5]
 
+`*` Please note that while acquisitions with `localizer` as their series description 
```

Comment:
@cmadjar 
this is where your PR #319 info conflict with the addition here.

Reply:
@MounaSafiHarab Thank you! I will edit this and try to push to this branch.

Reply:
cool, so I will merge now PR #319 then.